service/mantle/fedex/FedexServices.xml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<services xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://moqui.org/xsd/service-definition-2.1.xsd">
+
+    <!--Default Authentication Service for Main services -->
+        <!--Takes Connection Details and returns requestMap-->
+        <service verb="set" noun="Connection">
+            <in-parameters>
+                <parameter name="method"/>
+                <parameter name="connectUrl"/>
+                <parameter name="contentType"/>
+                <parameter name="requestMap" type="Map"/>
+                <parameter name="requestPath"/>
+                <parameter name="apiTokenType"/>
+                <parameter name="apiTokenValue"/>
+            </in-parameters>
+            <out-parameters>
+                <parameter name="responseMap"/>
+            </out-parameters>
+            <actions>
+
+                <log message="==============Connection Details: METHOD: ${method}, CONNECTION URL: ${connectUrl}, CONTENT TYPE: ${contentType} REQUEST MAP: ${requestMap} REQUEST MAP: ${requestPath} TOKEN TYPE: ${apiTokenType} TOKEN VALUE:${apiTokenValue}=============="/>
```

Comment:
Please add log message properly, This kind of log message not accepted.

Never print the confidential information like token, userid/password in log.